README.rst

```diff
@@ -555,7 +556,7 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **simplified_null_return**
-   | A return statement wishing to return nothing should be simply "return".
+   | A return statement wishing to return "void" should not return "null".
```

Comment:
the current description makes no sense, because:
1 - you cannot return `nothing` (unless you have class named like that), so a statement cannot wish to return it
2 - we do not check the willingness/wish of the statement to return 'nothing'